dbus/dbus.go

```diff
@@ -60,6 +61,27 @@ func PathBusEscape(path string) string {
 	return string(n)
 }
 
+// PathBusUnescape is the inverse of PathBusEscape.
+func PathBusUnescape(path string) string {
```

Comment:
Do you need this to be public in order to use it in your consuming application? If not, I'd prefer to keep it private for the moment, we can always export it later if required.

Reply:
Nope, will make private.